assets/js/components/OverlayCard/index.js

```diff
@@ -0,0 +1,104 @@
+/**
+ * OverlayCard component.
+ *
+ * Site Kit by Google, Copyright 2025 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import { Slide } from '@material-ui/core';
+import classnames from 'classnames';
+import PropTypes from 'prop-types';
+
+/**
+ * Internal dependencies
+ */
+import { BREAKPOINT_SMALL, useBreakpoint } from '../../hooks/useBreakpoint';
+import MainBody from './MainBody';
+
+export default function OverlayCard( {
+	className,
+	title,
+	description,
+	ctaButton,
+	dismissButton,
+	GraphicDesktop,
+	GraphicMobile,
+	visible,
+} ) {
+	const breakpoint = useBreakpoint();
+
+	if ( ! visible ) {
+		return null;
+	}
+
+	const classes = classnames( 'googlesitekit-overlay-card', className );
+
+	if ( breakpoint === BREAKPOINT_SMALL ) {
+		return (
+			<div className={ classes }>
```

Comment:
This `<div ...>` should be moved to the `MainBody` component.

Reply:
I found that moving this broke the `Slide` slide up functionality as it expects the passed element to be able to accept a ref. Fixing this issue makes the component more complex than keeping the div here in the main component.

Reply:
Ok, let it be here then.